.rubocop.yml

```diff
@@ -208,3 +208,27 @@ Style/CaseEquality:
 
 Style/CaseLikeIf:
   Enabled: true
+
+Style/ClassCheck:
+  Enabled: true
+
+Style/ClassEqualityComparison:
+  Enabled: true
+
+Style/ClassMethods:
+  Enabled: true
+
+Style/CollectionCompact:
```

Comment:
The commit summary says this enables CollectionMethods. Is that just a typo?

Reply:
I think that's just a typo; I'll fix it up.